core-tests/src/test/java/org/mule/runtime/core/internal/util/store/PersistentObjectStorePartitionTestCase.java

```diff
@@ -94,6 +95,27 @@ public void skipAndMoveCorruptedOrUnreadableFiles() {
     }
   }
 
+  @Test
+  public void skipAndMoveCorruptedOrUnreadableFilesWithoutCreatingDir() {
+    final String KEY = "key";
+    final String VALUE = "value";
+    try {
+      File.createTempFile("temp", ".obj", objectStoreFolder.getRoot());
+      File corruptedFolder = openDirectory(workingDirectory.getAbsolutePath()
+          + File.separator + PersistentObjectStorePartition.CORRUPTED_FOLDER);
+      deleteDirectory(corruptedFolder);
+
+      partition.store(KEY, VALUE);
+      // Expect the new stored object, and the partition-descriptor file
+      assertEquals(2, objectStoreFolder.getRoot().listFiles().length);
+
+      // Expect to have one more corrupted file in the corrupted folder
+      assertEquals(1, corruptedFolder.list().length);
+    } catch (Exception e) {
+      fail("Supposed to have skipped corrupted or unreadable files");
```

Comment:
hides the exception. Also no need to fail.. just let it be... let i be.. whispering words of wisdom.. let it be...